app/models/validations/sales/sale_information_validations.rb

```diff
@@ -237,16 +228,22 @@ def check_staircasing_non_socialhomebuy_mortgage(record)
   end
 
   def validate_mortgage_used_dont_know(record)
-    return unless record.mortgageused == 3
+    return unless record.mortgage_use_unknown?
 
     if record.discounted_ownership_sale?
-      record.errors.add(:mortgageused, I18n.t("validations.invalid_option", question: "Was a mortgage used for the purchase of this property?"))
+      record.errors.add(:mortgageused, I18n.t("validations.invalid_option", question: "was a mortgage used for the purchase of this property?"))
     end
     if record.outright_sale? && record.saledate && !record.form.start_year_after_2024?
-      record.errors.add(:mortgageused, I18n.t("validations.invalid_option", question: "Was a mortgage used for the purchase of this property?"))
+      record.errors.add(:mortgageused, I18n.t("validations.invalid_option", question: "was a mortgage used for the purchase of this property?"))
+      record.errors.add(:saledate, I18n.t("validations.financial.mortgage_used.year"))
+    end
+    if record.shared_ownership_scheme? && record.is_not_staircasing?
+      record.errors.add(:mortgageused, I18n.t("validations.invalid_option", question: "was a mortgage used for the purchase of this property?"))
+      record.errors.add(:staircase, I18n.t("validations.financial.mortgage_used.staircasing"))
     end
-    if record.shared_ownership_scheme? && record.staircase && record.staircase != 1
-      record.errors.add(:mortgageused, I18n.t("validations.invalid_option", question: "Was a mortgage used for the purchase of this property?"))
+    if record.stairowned && !record.stairowned_100?
+      record.errors.add :stairowned, I18n.t("validations.sale_information.stairowned.mortgageused_dont_know")
+      record.errors.add :mortgageused, I18n.t("validations.sale_information.stairowned.mortgageused_dont_know")
```

Comment:
Can there be consistency on whether to use brackets for `record.errors.add` please?

Not sure that I care which to go for tbh, it's just particularly odd to see both ways within one function.

Reply:
oh yes. that's quite yucky, not sure how I've ended up doing that

Reply:
ah I see I've copied them in from the other method when I combined them and not noticed they were different format